cmd/protoc-gen-loggingtags/Makefile

```diff
@@ -0,0 +1,43 @@
+TOP_LEVEL = $(shell git rev-parse --show-toplevel)
```

Comment:
Should the Makefile be rolled into the toplevel Makefile or kept separate as here?

Reply:
Couple things I'm thinking about: this would be the second command in x, behind s3env, and it seems likely most of what happens here would need to happen in each project that wants to use protoc-gen-loggingtags. Perhaps we should think about how we expect people to build and use this command?

Reply:
Assuming a project is already using gRPC and protobufs, what's required is that:

* `protoc-gen-loggingtags` is installed into your path (or the path to it is provided to `protoc`)
* The `safe.proto` file is available somewhere in source form, such that one of the paths passed to protoc as a `--proto_path` / `-I` flag + the path specified in the `import` statement in the proto file resolve to the `safe.proto` file. For example, `--proto_path=$GOPATH/src` and `import "github.com/heroku/x/a/b/c/safe.proto` or `--proto_path=$GOPATH/src/github.com/heroku/x` and import `a/b/c/safe.proto`.

I'd guess that users would vendor the command and install locally, which is presumably the same thing they're already doing for other tools.

Reply:
That all sounds good! I guess we need to get any testing running in CI and break out any bits around using in a project into docs?

Reply:
Okay, reflecting on all of this, I think we could go with something like:

* Detailed docs in cmd/x/protoc-gen-loggingtags
* Move the proto into a toplevel dir, so imports are simplified to `github.com/heroku/x/loggingtags/safe.proto`
* Move the build process into the toplevel Makefile
* Add brief info to toplevel README that links to the cmd's detailed docs

How does that sound?

Reply:
Sounds good!